.github/workflows/connector_ops_ci.yml

```diff
@@ -21,3 +22,33 @@ jobs:
         run: pip install --quiet -e ./tools/ci_connector_ops
       - name: Check test strictness level
         run: check-test-strictness-level
+  check-review-requirements:
+    name: "Check if a review is required from Connector teams"
+    runs-on: ubuntu-latest
+
+    if: ${{ github.repository == 'airbytehq/airbyte' }}
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+      - name: Install ci-connector-ops package
+        run: pip install --quiet -e ./tools/ci_connector_ops
+      - name: Write review requirements file
+        id: write-review-requirements-file
+        run: write-review-requirements-file >> $GITHUB_OUTPUT
+      - name: Get mandatory reviewers
+        id: get-mandatory-reviewers
+        run: print-mandatory-reviewers >> $GITHUB_OUTPUT
+      - name: Check if the review requirements are met
+        if: steps.write-review-requirements-file.outputs.REVIEW_REQUIREMENTS_FILE != ''
+        uses: Automattic/action-required-review@v3
+        with:
+          status: ${{ steps.get-mandatory-reviewers.outputs.MANDATORY_REVIEWERS  }}
```

Comment:
To be clear, generating and outputting this separately is a requirement of the github action, right? We can't generate that into `connector_org_review_requirements.yaml`?

Reply:
Exactly, I added the  `MANDATORY_REVIEWERS` to conveniently display a detailed status like `A review is required from teams: A, B, C` instead of the default `Review required` status. But all the essential information is captured in the `REVIEW_REQUIREMENTS_FILE` whose format is defined by the external action.

Reply:
Cool, I figured. Strange that you can't configure different status messages for the different sets of requirements in the action 😄